tests/utils.py

```diff
@@ -64,4 +66,4 @@ def read_yaml_test_data(root_path):
 
 def compare_yaml(config: Config, fixture_yaml: str) -> None:
     generated_yaml = config_to_yaml(config)
-    assert generated_yaml == fixture_yaml
+    assert generated_yaml == fixture_yaml, diff(generated_yaml, fixture_yaml)
```

Comment:
`testfixtures.diff` seems to be a [one-liner wrapper](https://github.com/simplistix/testfixtures/blob/master/testfixtures/comparison.py#L26-L41) for Python standard library `difflib.unified_diff`

do you feel like we need the whole `testfixtures` just for that?

Reply:
no, I'll refactor it back down

Reply:
looking good!